projects/scion/workbench-application.angular/src/lib/workbench-application.module.ts

```diff
@@ -17,6 +17,7 @@ import { ActivatorService } from './activator.service';
 import { DefaultMessageBus, ManifestRegistryService, MessageBoxService, MessageBus, NotificationService, Platform, PopupService } from '@scion/workbench-application.core';
 import { ForRootGuardService } from './for-root-guard.service';
 import { WorkbenchPopupOpenActivityActionDirective } from './workbench-popup-open-activity-action.directive';
+import { IntentService } from '@scion/workbench-application.core/lib/intent-service';
```

Comment:
You access some internal API here. Did you forget to export the service in `public_api.ts` in `workbench-application.core`? Should be: `import { IntentService } from '@scion/workbench-application.core';`

Also, you must re-export the service from `workbench-application.angular` to be used by Angular client applications without having to import `workbench-application.core`.